West midlands | September 2025 | Georgina Rogers | Sprint 1 | Form Controls

zilinskyte opened the pull request:
## Learners, PR Template

Self checklist

- [x ] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [ x] My changes meet the requirements of the task
- [ x] I have tested my changes
- [ x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

Form control HTML

## Questions

Ask any questions you have for your reviewer.

illicitonion commented on css-wireframe:
```diff
@@ -0,0 +1,106 @@
+  gap: 20px;
+}
+
+/* Articles */
```
What value are these comments giving beyond what the CSS selectors are providing?

illicitonion reviewed (commented):
This is generally looking really good - there are a few small things to pick up:
1. We're expecting you to store this in `Form-Controls/index.html`, but it looks like you made a new file `form.html`? Can you rename it?
Also, there are extra files in the root of the repo: `css-wireframe`, `form-control`, `html-wireframe` - can you delete those?
2. Looking at your page, there's an extra `<` after Black?
3. I left a question about CSS comments, but it's not actually for this exercise, so something to think about but not something you need to address here, as you will hopefully delete that file :)

illicitonion commented:
Thanks!

> 1. We're expecting you to store this in `Form-Controls/index.html`, but it looks like you made a new file `form.html`? Can you rename it?

The form-controls exercise is now in the right file ✅ 

> Also, there are extra files in the root of the repo: `css-wireframe`, `form-control`, `html-wireframe` - can you delete those?

By the "root" of the repo I meant the top-level directory. i.e. if you look in https://github.com/CodeYourFuture/Module-Onboarding (or your filesystem) you will see a readme.md and a netlify.toml and a LICENSE.txt and some other files in that directory - that's the root of the repository. The form-controls index.html is not in the root, it's in a subdirectory (named Form-Controls).

And sorry, my comment was really unclear/imprecise here. By "delete" I meant "undo the changes" rather than "delete the files" - it looks like you've deleted the files, which I didn't mean! In general in a pull request, we want to leave all of the files not related to our change alone, and only changes the files relevant to our change.

Fortunately, when we delete things in git, they are not gone forever, we can get them back! For each file you want to bring back, we can "check out" that file "like it was in some other version". The "main" branch is a version before you made any changes, so we can check out those files like they were on the main branch. You can do this by running:

```bash
git checkout main -- list of files you want to check back out
```

e.g. if you wanted to undo the changes to `.gitignore` and the `Wireframe` directory, you could run:

```bash
git checkout main -- .gitignore Wireframe
```

Hopefully you can use this to work out how to make it so that your PR changes exactly one file (`Form-Controls/index.html`) :)

> 2. Looking at your page, there's an extra `<` after Black?

I don't think you fixed this one up? See this screenshot for what I mean:

<img width="311" height="100" alt="Screenshot 2025-10-06 at 12 06 18" src="https://github.com/user-attachments/assets/b4c856b6-8d7c-433c-a5b1-66a8dcb78d6c" />

zilinskyte commented:
> Thanks!
> 
> > 1. We're expecting you to store this in `Form-Controls/index.html`, but it looks like you made a new file `form.html`? Can you rename it?
> 
> The form-controls exercise is now in the right file ✅
> 
> > Also, there are extra files in the root of the repo: `css-wireframe`, `form-control`, `html-wireframe` - can you delete those?
> 
> By the "root" of the repo I meant the top-level directory. i.e. if you look in https://github.com/CodeYourFuture/Module-Onboarding (or your filesystem) you will see a readme.md and a netlify.toml and a LICENSE.txt and some other files in that directory - that's the root of the repository. The form-controls index.html is not in the root, it's in a subdirectory (named Form-Controls).
> 
> And sorry, my comment was really unclear/imprecise here. By "delete" I meant "undo the changes" rather than "delete the files" - it looks like you've deleted the files, which I didn't mean! In general in a pull request, we want to leave all of the files not related to our change alone, and only changes the files relevant to our change.
> 
> Fortunately, when we delete things in git, they are not gone forever, we can get them back! For each file you want to bring back, we can "check out" that file "like it was in some other version". The "main" branch is a version before you made any changes, so we can check out those files like they were on the main branch. You can do this by running:
> 
> ```shell
> git checkout main -- list of files you want to check back out
> ```
> 
> e.g. if you wanted to undo the changes to `.gitignore` and the `Wireframe` directory, you could run:
> 
> ```shell
> git checkout main -- .gitignore Wireframe
> ```
> 
> Hopefully you can use this to work out how to make it so that your PR changes exactly one file (`Form-Controls/index.html`) :)
> 
> > 2. Looking at your page, there's an extra `<` after Black?
> 
> I don't think you fixed this one up? See this screenshot for what I mean:
> 
> <img alt="Screenshot 2025-10-06 at 12 06 18" width="311" height="100" src="https://private-user-images.githubusercontent.com/1131704/497773943-b4c856b6-8d7c-433c-a5b1-66a8dcb78d6c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTk3OTM1MTMsIm5iZiI6MTc1OTc5MzIxMywicGF0aCI6Ii8xMTMxNzA0LzQ5Nzc3Mzk0My1iNGM4NTZiNi04ZDdjLTQzM2MtYTViMS02NmE4ZGNiNzhkNmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTAwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTEwMDZUMjMyNjUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODFlNTIzMmE4ZmRhZmNkNGU1NmVjNDA3OTY1ODkyZjIwOWE2YzBlYmNkNWY2ODUyNjA1OTY3ZTA5OWYzZGFmYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.WRLRXsy25XV3L7aRe0mAopqthbgJNqfn4uGfKmoGFMw">

Hi. I believe I have readded all the files i deleted so now there is only one file changed in the PR. I hope the repository and subrepository are correct now - I've had issues as when I first started I didn't understand branches well and did everything in one... If this is still incorrect please can we set up a teams call so I make sure I understand what to change. I have removed the tag now (definitely!) Thank you for your help and comments.
